Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1889,9 +1983,12 @@ bool NetPlayClient::PollLocalPad(const int local_pad, sf::Packet& packet)
 
 void NetPlayClient::SendPadHostPoll(const PadIndex pad_num)
 {
-  if (!m_local_player->IsHost())
+  if (m_local_player->pid != m_current_golfer)
```

Comment:
Is `m_local_player->pid` zero for the server when not golfing? Otherwise non-golf mode might be broken I guess?

Reply:
No it's 1, but non-golf mode just disables the ability to change the golfer, so `m_current_golfer` is always 1.

Reply:
Whoops, overlooked that one, thanks! 👍

Reply:
I made it so `m_current_golfer` is initialized to 1 so it's slightly more obvious.